_maps/map_files/generic/CentCom_skyrat_z2.dmm

```diff
@@ -3966,7 +3966,7 @@
 /area/centcom/holding/cafedorms)
 "aMI" = (
 /obj/machinery/vending/hydronutrients{
-	products = list(/obj/item/reagent_containers/glass/bottle/nutrient/ez = 50, /obj/item/reagent_containers/glass/bottle/nutrient/l4z = 20, /obj/item/reagent_containers/glass/bottle/nutrient/rh = 10, /obj/item/reagent_containers/spray/pestspray = 30, /obj/item/reagent_containers/syringe = 5, /obj/item/storage/bag/plants = 30, /obj/item/cultivator = 10, /obj/item/shovel/spade = 10, /obj/item/plant_analyzer = 10)
+	products = list(/obj/item/reagent_containers/glass/bottle/nutrient/ez=50,/obj/item/reagent_containers/glass/bottle/nutrient/l4z=20,/obj/item/reagent_containers/glass/bottle/nutrient/rh=10,/obj/item/reagent_containers/spray/pestspray=30,/obj/item/reagent_containers/syringe=5,/obj/item/storage/bag/plants=30,/obj/item/cultivator=10,/obj/item/shovel/spade=10,/obj/item/plant_analyzer=10)
```

Comment:
Why is that even all var-edited in the first place.

Reply:
I have no clue, but apparently maptools doesn't like them having spaces there